core/src/main/java/org/sonarsource/sonarlint/core/telemetry/TelemetryClient.java

```diff
@@ -32,17 +35,21 @@
 import java.nio.file.Path;
 
 public class TelemetryClient {
-  private static final String TELEMETRY_ENDPOINT = "https://chestnutsl.sonarsource.com";
   private static final String TELEMETRY_PATH = "telemetry";
-  private static final int TELEMETRY_TIMEOUT = 30_000;
   private static final int MIN_HOURS_BETWEEN_UPLOAD = 5;
 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "TELEMETRY_TIMEOUT" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)